packages/hurl/src/html/unescape.rs

```diff
@@ -470,7 +470,7 @@ mod tests {
             check(e, e)
         }
 
-        // Check that non-existent named entities are returned unchanged
+        // Check that nonexistent named entities are returned unchanged
```

Comment:
US English prefers this spelling, if your project isn't using US English, then it might make sense to drop this.